app/assets/javascripts/directives/task_item.js

```diff
@@ -41,6 +41,17 @@ angular.module("OpenHq").directive("taskItem", function($rootScope, Task, Confir
           $scope.task.update();
         }
       });
+
+      $(".tasks ul.sortable").sortable({
```

Comment:
yeah... thats the way it was before though - i think it automatically clears any old sortable stuff when you run it again. Will look into it.